add_stream_error.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+
+class AddStreamError(Enum):
```

Comment:
Will this ever be used outside of `main` ?  Otherwise perhaps we can either:
1. place it in `main`
2. change the filename to `enums.py` to indicate that it's ok to store future enums in this file as well? 

But it's only a nitpick, so I'll leave this decision up to you 👍